examples/tutorials/api/models.py

```diff
@@ -237,7 +237,7 @@
 
 ######################################################################
 # Again you can check the `SPATIAL_MODELS` registry to see which models
-# are available or take a look at the :ref:`spatial-models-gallery`
+# are available or take a look at the :ref:`spatial_models_gallery`
```

Comment:
this does not link properly in my deployed docs

Reply:
might be related to the way the anchor is defined in the `...models/spectral/README.rst`?

Reply:
I have tried different ways of defining the anchor to corresponding sections but I keep getting `WARNING: undefined label: spectral_models_gallery` (also for spatial models) and cross-reference link are not properly set, although syntax seems to be fine. Anyway, go ahead with the merging. It can be tried to be fixed later.